lib/active_merchant/billing/gateways/axcessms.rb

```diff
@@ -103,7 +103,7 @@ def parse_element(response, node)
       end
 
       def build_request(payment_code, money, payment, options)
-        xml = Builder::XmlMarkup.new :indent => 2
+        xml = Builder::XmlMarkup.new indent: 2
         xml.instruct!
         xml.tag! 'Request', 'version' => API_VERSION do
```

Comment:
Nothing to do with this PR, but all this `tag!` stuff in here might be a good target for the next reFactory.

Reply:
Oh, definitely! I was noticing just how much XML still lives in Active Merchant going through all of these changes.